apps/app/src/app/components/hint/hint.component.scss

```diff
@@ -0,0 +1,35 @@
+@import 'variables';
+.hint {
+  background-color: var(--stf-gray-background);
+  padding: rythm(0.5);
+  ::ng-deep {
+    pre {
+      overflow: scroll;
+    }
+    a {
+      color: $gray-black;
+      font-weight: var(--stf-font-weight-semibold);
+    }
+    a:hover {
+      color: var(--stf-primary);
+    }
+  }
+}
+.hint-title {
+  cursor: pointer;
+  display: flex;
+  align-items: center;
+  color: $gray-contrast;
+  svg-icon {
+    transition: transform 0.2s;
```

Comment:
always better to specify a transition function like `transition: transform 0.2s ease-in-out;`